numismatic/feeds/base.py

```diff
@@ -156,6 +165,7 @@ class WebsocketClient(abc.ABC):
     websocket_url = None
     websocket = attr.ib(default=None)
 
+    # FIXME: Should this not be named subscribe?
```

Comment:
Yes, it should be named subscribe. However, on the command line, typing subscribe is way longer than typing listen.